tests/integration_tests/test_llm.py

```diff
@@ -531,16 +531,18 @@ def _verify_lm_lora_finetuning_layers(
             {POSTPROCESSOR: {MERGE_ADAPTER_INTO_BASE_MODEL: False}},
             id="adalora_not_merged",
         ),
-        pytest.param(
-            "adaption_prompt",
-            {},
-            id="adaption_prompt-defaults",
-        ),
-        pytest.param(
-            "adaption_prompt",
-            {"adapter_len": 6, "adapter_layers": 1},
-            id="adaption_prompt-modified-defaults",
-        ),
+        # TODO: <Alex>02/21/2024: Disabling AdaptionPrompt (waiting for PEFT release to fix
```

Comment:
I think in addition to commenting out the tests, we should also unregister the `adaption_prompt` adapter.